content/posts/2016-02-01_february-newsletter.md

```diff
@@ -0,0 +1,132 @@
+Title: February Newsletter
+Date: 2016-02-01
+Tags: newsletter
+Slug: February-newsletter-2016
+Author: Alain Domissy
+
+
+Pythonistas,
+
+
+Upcoming events
+---------------
+
+----
+
+
+[Saturday study group][saturday-meetup] (meetup.com) <br />
+San Diego PyLadies and SD Python are continuing the Saturday study group
+at the Ansir Innovation Center (12:30pm - 3pm) The goal of the Saturday study group is to
+learn Python together in a collaborative environment. The study group is open
+to people of all skill levels and is particularly open to those new to Python.
+Bring a laptop!
+
+[saturday-meetup]: http://www.meetup.com/pythonsd/events/227667838/
+
+
+[PyLadies Casual Study Group][] (meetup.com) <br />
+San Diego PyLadies are meeting at Eclipse Chocolat (now a usual location!!)
```

Comment:
We can probably drop the 'now a usual location!!' bit. They've been there for the past 7 or 8 of their Tuesday meetings.